src/App.tsx

```diff
@@ -17,18 +77,33 @@
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter
+                status={filterStatus}
+                searchTerm={searchTerm}
+                onFilterChange={handleFilterChange}
+                onSearch={handleSearch}
+                onClearSearch={handleClearSearch}
+              />
             </div>
 
-            <div className="block">
+            {isLoading ? (
               <Loader />
```

Comment:
Verify that the `Loader` component is correctly implemented and imported to display the loading state appropriately.